src/components/Table/Table.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useEffect } from 'react';
+import { Person } from '../../types';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+
+export const Table: React.FC = () => {
+  const { personSlug } = useParams();
+  const [requestState, setRequestState] = useState('loading');
+  const [peopleTable, setPeopleTable] = useState<Person[] | null>(null);
```

Comment:
I don't see the point of passing a _null_ value here instead of simply using an empty array as initial value